pkg/cmd/server/api/v1/types.go

```diff
@@ -765,7 +765,7 @@ type RFC2307Config struct {
 
 type ActiveDirectoryConfig struct {
 	// AllUsersQuery holds the template for an LDAP query that returns user entries.
-	AllUsersQuery LDAPQuery `json:"allUsersQuery"`
+	AllUsersQuery LDAPQuery `json:"usersQuery"`
```

Comment:
@deads2k We don't have a `userUIDAttribute` here, and we need one. I think (?) we assume `dn` right now? I don't really know why this is working without this field.